includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```
@@ -1875,39 +1869,145 @@ public function get_login_confirmation_settings() {
+			$locations = $settings[ $legacy_keys[ $method ] ] ?? [];
+			if ( ! is_array( $locations ) ) {
+				continue;
+			}
+
+			foreach ( $locations as $location ) {
+				$map[ $location ][] = $method;
+			}
+		}
+
+		return $map;
+	}
```

PHPMD (2.15.0)

[error] The method build_locations_map_from_legacy is not named in camelCase. (undefined)

(CamelCaseMethodName)